LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-    MIT License
+    The MIT License (MIT)
```

Comment:
This doesn't seem right. According to openlicenses.org, the full text should be in the file.

Reply:
Do you have a better link. `openlicenses.org` doesn't resolve to anything for me. 
Also, .NET, Java, and JS have it as `The MIT License (MIT)`. But Python has it as `MIT License`. 
I went with the option I saw more.

Reply:
If that's what the others are doing, I just hope legal cleared it. That's not typical. And the site is actually https://opensource.org/li. Sorry, my bad.

That said, the text is right in the license:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.